crates/viewer/re_view_spatial/src/picking.rs

```diff
@@ -39,7 +39,9 @@ pub struct PickingRayHit {
 
 #[derive(Clone, PartialEq)]
 pub struct PickingResult {
-    /// Picking ray hits. NOT sorted by distance but rather by source of picking.
+    /// Picking ray hits.
+    ///
+    /// The hits are ROUGHLY sorted front-to-back, i.e. closest hits are first.
```

Comment:
thank you!
My expectation is that it works correctly the way you coded it up here. Those images show up in the gpu & rectangle results, but then get filtered out of the gpu results. And since you put the rectangles always lower it works